packages/material-ui-codemod/README.md

```diff
@@ -1067,6 +1068,39 @@ npx @material-ui/codemod@next v5.0.0/with-width <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#material-ui-core-styles).
 
+#### `mui-replace`
+
+Replaces all `@material-ui` related prefix with `@mui`
+
+```diff
+- import Alert from '@material-ui/core/Alert';
++ import Alert from '@mui/material/Alert';
+- import { makeStyles } from '@material-ui/styles';
++ import { makeStyles } from '@mui/styles';
+- import { SxProps } from '@material-ui/system';
++ import { SxProps } from '@mui/system';
+- import { deepmerge } from '@material-ui/utils';
++ import { deepmerge } from '@mui/utils';
+- import Mansory from '@material-ui/lab/Mansory';
++ import Mansory from '@mui/lab-material/Mansory';
+- import { DistributiveOmit } from '@material-ui/types';
++ import { DistributiveOmit } from '@mui/types';
+
```

Comment:
Usually, git diff groups the removals and the additions. Here, we alternative them. Maybe it's the sign that we need multiple ```diff sections?

Reply:
updated.